coralproject/talk
# Updated French translations : stream.ftl, account.ftl and framework.ftl

## What does this PR do?




Update the French translations for the user front-end of Coral. No modification has been made to admin interface labels or the authentication workflow labels:
- Added missing translations
- Fixed typos in some variables
- Adapt the translation to French specificities. You should think about splitting fr-FR fr-BE and fr-CA (or more) to allow local adaptations

## What changes to the GraphQL/Database Schema does this PR introduce?




None

## How do I test this PR?


Deploy Coral, add the JS to a test web env, set your locale to French and check the labels.
